lib/vscode/package.json

```diff
@@ -106,7 +104,6 @@
     "@types/graceful-fs": "4.1.2",
     "@types/gulp-postcss": "^8.0.0",
     "@types/http-proxy-agent": "^2.0.1",
-    "@types/keytar": "^4.4.0",
```

Comment:
Note for review: `@types/keytar` is deprecated since `keytar` includes types - I removed both and added a `typings` file.